core/test/unit/models/role_spec.js

```diff
@@ -0,0 +1,32 @@
+const models = require('../../../server/models');
+const ghostBookshelf = require('../../../server/models/base');
+const testUtils = require('../../utils');
+const should = require('should');
+
+describe('Unit: models/role', function () {
+    before(function () {
+        models.init();
+    });
+
+    before(testUtils.teardown);
```

Comment:
Is it deffo necessary to teardown the entire db for this test?

Reply:
🤷‍♂️ I was only following the other tests I saw. In previous projects I've worked on the db was always torn for each test otherwise it was very easy to introduce interdependencies that were hard to debug because errors start showing up in unrelated tests

Reply:
Yeah fair enough - I don't actually know either :grin:. I've always added new objects to db and removed them after the test to avoid the overhead - but no biggie if we already to like this!